phase1/openstack/configure-vm-kubeadm-master.sh

```diff
@@ -0,0 +1,6 @@
+
+OPTS=''
+if [[ -n "${k8s_kubernetes_version}" ]]; then
+  OPTS="--kubernetes-version ${k8s_kubernetes_version}"
+fi
+kubeadm init --token "${k8s_kubeadm_token}" --apiserver-bind-port 443 --skip-preflight-checks --apiserver-cert-extra-sans="${k8s_advertise_addresses}" $OPTS
```

Comment:
This is to be in sync with GCE, most of the kubeadm related stuff is exactly same as what is used in GCE. we should ideally segregate the kubeadm related stuff to common reusable scripts instead of duplicating the way i did in this PR. I would take it up in subsequent PR, if it is OK.